config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -260,3 +260,51 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  kubernetes/perf-tests:
+  - name: pull-perf-tests-clusterloader
```

Comment:
Let's name it clusterloader2 to avoid confusion with https://github.com/kubernetes/perf-tests/tree/master/clusterloader